CONTRIBUTING.md

```diff
@@ -1,13 +1,54 @@
-#Making Database Changes
+#Contributing to Qiita
+
+Qiita is an open source software package, and we welcome community contributions. You can find the source code and test code for Qiita under public revision control in the Qiita git repository on [GitHub](http://github.com/biocore/qiita). We very much welcome contributions.
+
+This document covers what you should do to get started with contributing to Qiita. You should read this whole document before considering submitting code to Qiita. This will save time for both you and the Qiita developers.
+
+#General Notes on Development
+
+Adding source code to Qiita, can take place in three different modules:
+
+* `qiita_pet`: Contains the graphical user interface layer of the system.
```

Comment:
Sure, done!

On (Mar-05-15| 4:04), Antonio Gonzalez wrote:
> > @@ -1,13 +1,54 @@
> > -#Making Database Changes
> > +#Contributing to Qiita
> > +
> > +Qiita is an open source software package, and we welcome community contributions. You can find the source code and test code for Qiita under public revision control in the Qiita git repository on [GitHub](http://github.com/biocore/qiita). We very much welcome contributions.
> > +
> > +This document covers what you should do to get started with contributing to Qiita. You should read this whole document before considering submitting code to Qiita. This will save time for both you and the Qiita developers.
> > +
> > +#General Notes on Development
> > +
> > +Adding source code to Qiita, can take place in three different modules:
> > +
> > +* `qiita_pet`: Contains the graphical user interface layer of the system.
> 
> Do you think is worth adding the languages each of this rely on? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/902/files#r25858670